### PR TITLE
EDLY-6859: Fix set password screen shows reset password text

### PR DESCRIPTION
**Pull request by Waleed-Mujahid**
**Description:** 
When a new user account is created, set password screen shows `reset your password` message. This PR updates the text.

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-6859

### CODE REVIEW

**Review by Anas-hameed — APPROVED** [review]
LGTM

File: openedx/core/djangoapps/user_authn/views/password_reset.py [hunk 1]
```diff
@@ -523,6 +523,7 @@ def post(self, request, *args, **kwargs):
     def dispatch(self, *args, **kwargs):
         self.uidb36 = kwargs.get('uidb36')
         self.token = kwargs.get('token')
+        self.isNewUser = kwargs.get('method', None) == 'new_password'
```
Comment (muhammadali286) [inline openedx/core/djangoapps/user_authn/views/password_reset.py:526]: In pyhton we use snake case naming convention
isNewUser -> is_new_user

**Review by muhammadali286 — CHANGES_REQUESTED** [review]

File: openedx/core/djangoapps/user_authn/views/password_reset.py (hunk 1 above)
Comment (Waleed-Mujahid) [reply openedx/core/djangoapps/user_authn/views/password_reset.py]: updated

**Review by Anas-hameed — APPROVED** [review]

File: lms/templates/registration/password_reset_confirm.html [hunk 2]
```diff
@@ -12,7 +12,12 @@
 <%namespace name='static' file='../static_content.html'/>
 
 <%block name="title">
-    <title>${_("Reset Your {platform_name} Password").format(platform_name=platform_name)}</title>
+    % if isNewUser:
+        <title>${_("Set Your {platform_name} Password").format(platform_name=platform_name)}</title>
+    % else:
+        <title>${_("Reset Your {platform_name} Password").format(platform_name=platform_name)}</title>
+    % endif
+
```
[the remark this answers is not in the document]
Comment (Waleed-Mujahid) [reply lms/templates/registration/password_reset_confirm.html]: Done

**Review by muhammadali286 — APPROVED** [review]

**Pull request merged**